README.md

```diff
@@ -11,53 +11,238 @@ Pour faire ce codelab, vous avez besoin des outils suivants :
 * NodeJS 8.x
 * un IDE (WebStorm, Visual Studio Code, Vim, ...)
 
-Pour vous aider, nous vous conseillond d'utiliser l'extention [**Vue-devtools**](https://github.com/vuejs/vue-devtools) disponible pour:
+Pour vous aider, nous vous conseillons d'utiliser l'extention [**Vue-devtools**](https://github.com/vuejs/vue-devtools) disponible pour:
 - [Chrome](https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd)
 - [Firefox](https://addons.mozilla.org/en-US/firefox/addon/vue-js-devtools/)
 
-## Exercices
-
-Chaques exercice pourra partir d'un code propre:
-
-- `step0`: un site statique que nous allons vuetifier
-- `step1`: projet tel qu'attendu après l'exercice 1
-- `step2`: projet tel qu'attendu après l'exercice 2
-- `step3`: projet tel qu'attendu après l'exercice 3
-- etc...
-
-### Exercice 1 - Getting started
-
-Génération d'un template vue avec vue-cli. Template webpack ou webpack-simple ?
-
-### Exercice 2 - Composants
-
-Création des premiers composants (fichier.vue) pour:
-- recherche
-- résultats
-
-Découverte de props, data, watch, et des hooks
+> Pour les utilisateurs de VS Code, nous vous conseillons d'utiliser l'extension [**vetur**](https://vuejs.github.io/vetur/)
 
-### Exercice 3 - Directives
+## Présentation
 
-Utilisation de v-if
-Utilisation de v-for
-Utilisation de v-model
-Utilisation de v-on
+La présentation accompagnant ce code labs est disponible [ici](https://docs.google.com/presentation/d/1tk_6xn5FNG0VB_nT-NWxga8gqoo9LT6R0wjiLMBopak/edit?usp=sharing)
 
-### Exercice 4 - Filtres
-
-Création d'un filter uppercase + date
-Découverte de computed properties
-
-### Exercice 5 - Ressources
-
-Axios + fetch des données
-
-### Exercice 6 - Routeur
-
-vue-router pour les routes favoris et recherche
-
-### Exercice 7 - VueX
+## Exercices
 
-favoris + recherche avec vuex
+> Chaques exercice pourra partir d'un code propre:

```

Comment:
I don't think we used to work this way, but it could be great to make this at the end. Was it the idea ?

Reply:
yes, I've created a branch named `PW` for that (only the first 3 exercices are included so far)